packages/reference/src/entries/WrappedEntryCard/FetchingWrappedEntryCard.tsx

```diff
@@ -57,21 +61,13 @@ async function openEntry(
 }
 
 export function FetchingWrappedEntryCard(props: EntryCardReferenceEditorProps) {
-  const { getEntry, getAsset, loadEntityScheduledActions, entries } = useEntities();
-
-  React.useEffect(() => {
-    getEntry(props.entryId);
-    // eslint-disable-next-line react-hooks/exhaustive-deps -- TODO: Evaluate the dependencies
-  }, [props.entryId]);
+  const { data: entry, status } = useEntity<Entry>('Entry', props.entryId);
+  const { getEntityScheduledActions } = useEntityLoader();
+  const loadEntityScheduledActions = () => getEntityScheduledActions('Entry', props.entryId);
```

Comment:
I think these `loadEntityScheduledActions` need to be wrapped in a `useMemo` to avoid re-rendering in children?